corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html

```diff
@@ -187,6 +187,14 @@ <h1 style="font-size: 1.8em;">{% trans "Case Summary" %}</h1>
     <li ng-repeat="question in questions">
         {% verbatim %}<i class="hq-icon fa-li {{ getIcon(question.question.type) }}"></i>{% endverbatim %}
         {% verbatim %}<span ng-show="showLabels">{{ getQuestionLabel(question.question) }}</span>{% endverbatim %}
+        {% verbatim %}
+        <ul ng-if="question.question.options">
+            <li ng-repeat="option in question.question.options">
+                {{ getSelectOptionsLabel(option) }}
```

Comment:
does this respect the question IDs toggle? was looking at the screenshot and just thinking IDs are often more useful than labels.